### PR TITLE
Updates to fix getCapturesForRecurring

### PR DESCRIPTION
**Pull request by TPeterson94070**

### CODE REVIEW

**Review by TPeterson94070 — COMMENTED** [review]
Also added back the unified USER definition string.

File: CwHelperC/src/org/cwepg/hr/TunerManager.java [hunk 1]
```diff
@@ -1467,6 +1467,7 @@ public ArrayList<Capture> getCapturesForRecurring(String channelName, Slot origi
             Capture aCapture = getCaptureForChannelNameSlotAndTuner(channelName, listedSlot, tunerString, protocol);
             if (aCapture != null){
                 captureList.add(aCapture);
+                aCapture.isRecurring = true;
```
Comment (sengsational) [inline CwHelperC/src/org/cwepg/hr/TunerManager.java:1470]: The preferred way to do this is  "aCapture.setRecurring(true);" At the moment, the two approaches cause the same result.  But in the future, maybe there's a second thing to do whenever that variable is changed.  The way to enforce this is to make the variable in the Capture class "private", which is probably the way I should have done it.

**Comment by sengsational** [comment]
If you're running the latest from my repository, you should be able include in this PR "aCapture.setRecurrenceDays(recurring);" right under where you added "aCapture.setRecurring(true);".

File: CwHelperC/src/org/cwepg/hr/TunerManager.java (hunk 1 above)
Comment (TPeterson94070) [reply CwHelperC/src/org/cwepg/hr/TunerManager.java:1470]: OK, do I need to modify my pull request or how is this handled?

Comment (sengsational) [reply CwHelperC/src/org/cwepg/hr/TunerManager.java:1470]: My expertise with managing changes through this PR mechanism is limited.  My advice would be to add to or somehow edit the PR to get everything in it.  The "right way" might be to edit the existing PR or to make a new one.  I really don't know.  I like that we're figuring this stuff out, though.

**Comment by TPeterson94070** [comment]
Ok, I mistook your original comment as simply a stylistic note rather than a request for my action. I'll edit or resubmit the PR. 


-------- Original Message --------

File: CwHelperC/src/org/cwepg/hr/TunerManager.java (hunk 1 above)
Comment (TPeterson94070) [reply CwHelperC/src/org/cwepg/hr/TunerManager.java]: Have made the suggested changes.  Will either edit this PR or close and resubmit.

**Comment by TPeterson94070** [comment]
It appears that my edited repository is already now part of this PR...so I'm done???

**Pull request closed without merging**